footer.php

```diff
@@ -22,14 +22,26 @@
 
 		<div class="site-info">
 			<div class="site-name">
-				<?php
-				$blog_info = get_bloginfo( 'name' );
-				if ( ! empty( $blog_info ) ) {
-					?>
-					<a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a>
+				<?php if ( has_custom_logo() ) : ?>
+					<?php if ( is_front_page() ) : ?>
+						<?php the_custom_logo(); ?>
+					<?php else : ?>
```

Comment:
Is this needed? The custom logo already has the theme support for unlinking it on the home page, has it been tested?
See https://github.com/WordPress/twentytwentyone/blob/trunk/functions.php#L91

Reply:
yeah I tested it but I didn't notice the double link… 🤦 

<img width="999" alt="Capture d’écran 2020-09-26 à 08 32 41" src="https://user-images.githubusercontent.com/1590998/94332169-024cee80-ffd3-11ea-9d0d-e644a4481899.png">

Reply:
Oops

Reply:
Apologies, I'm not quite following — what needs to happen here? I understand that the double link should be removed. But is the idea to only show the custom logo if it's the front page?

Reply:
The idea is to show the logo if there is a logo. Show the title if there is no logo.

Neither the title or the logo should be linked on the front page.

But the logo already has a custom link built in, and it does not need to be altered because there is already theme support for unlinking the logo.